aws-testing/utils_neuron.py

```diff
@@ -582,11 +614,19 @@ def build_grid_space_qwen3_235b(self):
         # TODO: consider removing DP replicas of groups and parallelize different tests on different cores if possible
         # Grid space for testing
         grid_space = []
-        # TODO add EP
-        for mesh_spec in [{"fsdp": -1, "model": 64}]:
-            batch = 4 if mesh_spec["model"] == 16 else 1
+        
```

Comment:
also add EP tests for the other build_grid_space functions like switch-xxl, llama, etc